gf-guide/metrics.md

```diff
@@ -134,40 +137,40 @@ Setting vertical metrics usually falls into the following two categories:
 
 Set these values to be the same across all masters to ensure that output instances have equal vertical metrics:
 
--   `typoAscender` and `hheaAscender` set higher than `À`
+-   `typoAscender` and `hheaAscender` set higher than `Ắ`
 -   `typoLineGap` and `hheaLineGap` set to `0`
--   `typoDescender` and `hheaDescender` set lower than the deepest descender of the primary script.
+-   `typoDescender` and `hheaDescender` set to visually leave Caps height centered in the line, but lower than the deepest descender of the primary script.
 -   `winAscent` and `winDecent` set to `yMax` and `yMin` (absolute highest and lowest point in the font)
 -   `use_typo_metrics` is enabled
 
-Expected result: vertical metrics should be around 120-130% of UPM. Anything greater, and the metrics may look too loose.
+Expected result: vertical metrics should be around 130% of UPM. Anything greater, and the metrics may look too loose.
 
 **Example**
 
 A new Latin family has the following qualities:
 
 -   UPM is `1000`
--   `yMax` of tallest `A grave` in the familly (bold for this example) = `940`
--   `yMin` of deepest a-z letter (`g` bold in this family) = `235`
--   Caps height (`H`or `Z` bbox height) = `730`
--   Family's `yMax = 1116` (Bold `A breve hookabove` for this family)
--   Family's `yMin = 320` (ExtraLight `c cedilla` for this family)
+-   `yMax` of tallest `A breve acute` in the familly (bold for this example) = `1000`
+-   `yMin` of deepest a-z letter (`g` bold in this family) = `210`
+-   Caps height (`H`or `Z` bbox height) = `697`
+-   Family's `yMax = 1000` (Black `A breve hookabove` for this family)
+-   Family's `yMin = -260` (ExtraLight `c cedilla` for this family)
 
 1.  Set the default values, following the schema above:
 
 ``` code
-typoAscender = 965 # (UPM * 1.2 - CapsHeight) / 2 + CapsHeight which is greater than Agrave, perfect.
-typoDescender = -235 # (UPM * 1.2 - CapsHeight) / 2 which is equal to deepest letterform
+typoAscender = 1000 # which is matches tallets `A breve acute` in the family.
+typoDescender = -303 # an equal or similar value added to the Caps Height which is greater than deepest letterform.
```

Comment:
I would keep the calculation changing 1.2 to 1.3, in that case typoAscender=1015 (which is good cause above Abreveacute).

Reply:
What I found risky about suggesting math calculations is many users will follow them blindly. And I've seen in many cases this will not directly work ok since the marks included are actually going beyond them. So I prefer providing info relative to each case and the maths just as a reference.